src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSSemanticConventions.cs

```diff
@@ -18,6 +18,7 @@ internal static class AWSSemanticConventions
     public const string AttributeAWSBedrockDataSourceId = "aws.bedrock.data_source.id";
     public const string AttributeAWSBedrockGuardrailId = "aws.bedrock.guardrail.id";
     public const string AttributeAWSBedrockKnowledgeBaseId = "aws.bedrock.knowledge_base.id";
+    public const string AttributeAWSeBedrock = "aws_bedrock";
```

Comment:
typo?  Should be `AttributeAWSBedrock`?

Reply:
Is this used anywhere?  (I don't have the full code base in front me)

Reply:
Yep it is used here https://github.com/muhammad-othman/opentelemetry-dotnet-contrib/blob/aws-instrumentation-fix-request-specific-info/src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs#L180

And the typo was fixed.